src/commands/base.ts

```diff
@@ -46,14 +51,16 @@ export default abstract class Base extends Command {
     token = ''
     projectKey = ''
     authPath = path.join(this.config.configDir, 'auth.yml')
-    configPath = path.join(this.config.configDir, 'config.yml')
+    configPath = path.join(this.config.configDir, 'user.yml')
+    repoConfigPath = '.devcycle/config.yml'
```

Comment:
shouldnt this be read from the repo-config-path flag? And have a default value set there?

Reply:
These are the default values. If the flags exist, they override these values later. (Lines 130-140)